pom.xml

```diff
@@ -113,6 +115,16 @@
       <artifactId>pipeline-stage-step</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
```

Comment:
what do you need this for? (if you do need it I don't think you need both mockito's

Reply:
TestResult is a final class so this dependency is needed to be able to mock it.